web/src/repogroups/cncf.tsx

```diff
@@ -0,0 +1,20 @@
+import { RepogroupMetadata } from './types'
+import * as React from 'react'
+
+export const cncf: RepogroupMetadata = {
+    title: 'CNCF',
+    name: 'cncf',
+    url: '/cncf',
+    description: (
+        <>
+            Search the{' '}
+            <a href="https://landscape.cncf.io/project=hosted" target="_blank" rel="noopener noreferrer">
```

Comment:
@daxmc99 is this a good link to surface the projects under CNCF? anything else you would suggest.

Reply:
This is likely fine.  There is a bit of a joke though that CNCF landscape is updated daily with new projects. Linking directly to it might only make sense if our repolist will grow dynamically. 
https://github.com/cncf/toc/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc there are new projects being added frequently

Reply:
Perhaps use `card` mode for a cleaner look 
```suggestion
            <a href="https://landscape.cncf.io/format=card-mode&project=hosted" target="_blank" rel="noopener noreferrer">
```

Reply:
@daxmc99 great to know this list changes often 😅 @attfarhan I think we should remove the number to avoid having to update this continuously.
Initially we can add new projects to the repo list based on requests from the CNCF community.